src/modules/project/pages1/projectDashboard/views/promote/GeyserPromotionSection.tsx

```diff
@@ -69,6 +73,38 @@ export const GeyserPromotionSection = ({
             "When enabled your project becomes eligible to be shared through Geyser's promotion network and partner digital media companies, bringing more eyes to your project to enable more contributions.",
           )}
         </Body>
+        <HStack spacing={4} mb={4}>
+          {[
+            {
+              light: 'https://storage.googleapis.com/geyser-projects-media/platform/bitcoin-news-light.png',
+              dark: 'https://storage.googleapis.com/geyser-projects-media/platform/bitcoin-news-dark.png',
+              alt: 'Bitcoin News logo',
+            },
+            {
+              light: 'https://storage.googleapis.com/geyser-projects-media/platform/lightning-news-light.png',
+              dark: 'https://storage.googleapis.com/geyser-projects-media/platform/lightning-news-dark.png',
+              alt: 'Lightning News logo',
+            },
+            {
+              light: 'https://storage.googleapis.com/geyser-projects-media/platform/bff-light.png',
+              dark: 'https://storage.googleapis.com/geyser-projects-media/platform/bff-dark.png',
+              alt: 'BFF logo',
+            },
+            {
+              light: 'https://storage.googleapis.com/geyser-projects-media/platform/bitcoin-bits-light.png',
+              dark: 'https://storage.googleapis.com/geyser-projects-media/platform/bitcoin-bits-dark.png',
+              alt: 'Bitcoin Bits logo',
+            },
+          ].map((logo) => (
+            <Image
+              key={logo.alt}
+              src={colorMode === 'light' ? logo.light : logo.dark}
```

Comment:
you can use useColorModeValue hook for this as well.  That makes it no need for string comparison.